app/constants/chartTypes.js

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
I would consider breaking out the selectableChartTypes object into multiple files with a root importer:

```
types
- index.js
- NVD3
--index.js
--pieChart.js
--discreteBarChart.js
--linechart.js
```

The individual files will have to exports:

```
export const config = {};
export const default = {};
```

I would also move the getChartTypeObject and get getChartTypeDefaultOpts to ~~service~~ util files instead of constant files.

IMO constant files should only contain constants.